pkg/minikube/cluster/commands.go

```diff
@@ -29,7 +29,7 @@ var stopCommand = "sudo killall localkube || true"
 
 var startCommandFmtStr = `
 # Run with nohup so it stays up. Redirect logs to useful places.
-sudo sh -c 'PATH=/usr/local/sbin:$PATH nohup /usr/local/bin/localkube %s --generate-certs=false --logtostderr=true --node-ip=%s > %s 2> %s < /dev/null &'
+sudo sh -c 'PATH=/usr/local/sbin:$PATH nohup /usr/local/bin/localkube %s --network-plugin=%s --container-runtime=%s --generate-certs=false --logtostderr=true --node-ip=%s > %s 2> %s < /dev/null &'
```

Comment:
Would you mind only inserting these new flags if the user has set them?

We have some alpha builds of localkube published that will crash if these flags are passed. See #512 for the bug tracking the main issue.

If this makes the formatting get a bit unwieldy it might be a good idea to switch to a text/template.